docs/aws/README.md

```diff
@@ -67,12 +74,13 @@ eksctl create iamidentitymapping \
   --group system:nodes
 ```
 
-### Install Karpenter
+Now, Karpenter can send requests for new EC2 instances to AWS. 
+
+## Install Karpenter Helm Chart
 
-Use [`helm`](https://helm.sh/) to deploy Karpenter to the cluster.
-For additional values, see [the helm chart values](https://github.com/awslabs/karpenter/blob/main/charts/karpenter/values.yaml)
+Use helm to deploy Karpenter to the cluster. 
 
-> We created a Kubernetes service account with our cluster so we don't need the helm chart to do that.
+We created a kubernetes service account when we created the cluster using eksctl. Thus, we don't need the helm chart to do that.
```

Comment:
I think we can probably leave out the second sentence starting with "Thus", as it refers to something historical that isn't. A reader following this guide need not be aware that you could have done this with helm in an alternative version of this guide.

Reply:
I think it's helpful to explain why we are setting certain options here, to connect the dots between the service account earlier and the option here. 

```
helm upgrade --install karpenter karpenter/karpenter \
  --namespace karpenter --set serviceAccount.create=false
```